cpp/code/basic_arrow.cc

```diff
@@ -63,3 +64,71 @@ arrow::Status ReturnNotOk() {
 TEST(BasicArrow, ReturnNotOkNoMacro) { ASSERT_OK(ReturnNotOkMacro()); }
 
 TEST(BasicArrow, ReturnNotOk) { ASSERT_OK(ReturnNotOk()); }
+
+/// \brief Sum numeric values across columns
+///
+/// Only supports floating point and integral types. Does not support decimals.
+class TableSummation {
+ public:
+  double partial;
```

Comment:
Is this me, or is it uninitialized? Also, this member doesn't need to be public AFAICT.
```suggestion
  double partial = 0.0;
 public:
```

Reply:
Ah I guess that is undefined behavior in C++ if it's not set. Thanks!